VERSION.txt

```diff
@@ -1 +1 @@
-6.6.0
+6.7.0
```

Comment:
Good question. It's a non-breaking feature addition though. Don't mind either way but my thinking was, feature addition, non-breaking, just do a min bump.